Card/src/main/java/com/paypal/android/card/APIClientError.kt

```diff
@@ -0,0 +1,57 @@
+package com.paypal.android.card
```

Comment:
Are these errors something that could be used across all SDK modules? Would it make more sense for them to live in the core module?